samples/dotnet/kernel-syntax-examples/Example35_DIContainer.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.AI.TextCompletion;
+using Microsoft.SemanticKernel.Connectors.AI.OpenAI.TextCompletion;
+using Microsoft.SemanticKernel.Memory;
+using Microsoft.SemanticKernel.Services;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Microsoft.SemanticKernel.TemplateEngine;
+using RepoUtils;
+
+/**
+ * The following examples show how to use SK SDK in applications using DI/IoC containers.
+ */
+public static class Example35_DIContainer
+{
+    public static async Task RunAsync()
+    {
+        await UseKernelInDIPowerAppAsync();
+
+        await UseKernelInDIPowerApp_AdvancedScenarioAsync();
+    }
+
+    /// <summary>
+    /// This example shows how to registered Kernel in DI container without registering it's dependencies and using KernelBuilder instead.
```

Comment:
``
```suggestion
    /// This example shows how to register a Kernel in a DI container using KernelBuilder instead of 
    /// registering its dependencies.
```